src/client.js

```diff
@@ -158,13 +158,31 @@ Client.prototype.connect = function(args) {
 	var irc = slate(stream);
 	identd.hook(stream, username);
 
-	if (args.password) {
-		irc.pass(args.password);
-	}
+
 
 	irc.me = nick;
-	irc.nick(nick);
-	irc.user(username, realname);
+	if (!!args.sasl) {
```

Comment:
Just do if(args.sasl)

Never use "!!" it's very confusing for people who don't know what it means (and there's no real advantage to it. In this case, args.sasl should only ever be a boolean, so theres no need to turn it into a boolean.